CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
-## 6.25.2. (January 31, 2023). Tested on Artifactory 7.49.6
+## 6.26.0. (January 31, 2023). Tested on Artifactory 7.49.6
 
 IMPROVEMENTS:
 
-* resource/artifactory_remote_*_repository: documentation, add a note for `propagate_query_params`. This attribute should be used only with Generic repo type.
+* resource/artifactory_remote_*_repository: `propagate_query_params` attribute is removed from the common remote repository configuration. This attribute only works with Generic repo type. This change is implemented in schema V2 and migrator was added. During the migration from V1 to V2 that attribute will be removed.
```

Comment:
We may want to include this https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/state-migration so our users can read up on what state migration means.

Reply:
Added.